src/Components/Books.js

```diff
@@ -1,42 +1,68 @@
+  const [author, setauthor] = useState('');
+
+  // consuming state
+  const books = useSelector((state) => state.books);
+  const dispatch = useDispatch();
+  const id = Math.random().toString();
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    dispatch(addedBook({ title, author, id }));
+  };
+
+  const handleDelete = (id) => {
+    dispatch(removeBook(id));
+  };
 
-export default function Books(props) {
-  const { title, author } = props;
   return (
     <>
       <div className="container">
-        <ul className="ul-books">
-          <li className="li-books">
-            <span>{title}</span>
-            <span>{author}</span>
-            <button type="submit">Remove</button>
-          </li>
-        </ul>
+        <div className="my-4 border border-3 p-0">
+          <div>
+            {books.map((book) => (
+              <div key={book.id} className="d-flex justify-content-between ms-0 p-3">
+                <span>
+                  Title:
+                  {' '}
+                  {book.title}
+                </span>
+                <span>
+                  Author:
+                  {' '}
+                  {book.author}
+                </span>
+                <button
+                  className="btn btn-primary mb-3"
+                  type="submit"
+                  onClick={() => { handleDelete(book.id); }}
+                >
+                  Remove
+
+                </button>
+              </div>
+            ))}
+          </div>
+        </div>
         <h2 className="fs-2 text">Add New Book</h2>
         <form className="row g-3">
-          <div className="col-8">
-            <input type="password" className="form-control" id="inputPassword2" placeholder="Book title" />
+          <div className="col-7">
+            <input type="text" className="form-control" value={title} onChange={(e) => settitle(e.target.value.slice(0, 1).toUpperCase() + e.target.value.slice(1, e.target.value.length).toLocaleLowerCase())} id="inputPassword2" placeholder="Book Title" />
           </div>
-          <div className="col-2">
-            <select className="custom-select form-control" id="inputGroupSelect04">
-              <option selected>Category</option>
-              <option value="1">One</option>
-              <option value="2">Two</option>
-              <option value="3">Three</option>
-            </select>
+          <div className="col-3">
+            <input type="text" className="form-control" value={author} onChange={(e) => setauthor(e.target.value.slice(0, 1).toUpperCase() + e.target.value.slice(1, e.target.value.length).toLocaleLowerCase())} id="inputPassword2" placeholder="Book Author" />
           </div>
           <div className="col-2">
-            <button type="submit" className="btn btn-primary mb-3">Add Book</button>
+            <button type="submit" className="btn btn-primary mb-3" onClick={handleSubmit}>Add Book</button>
           </div>
         </form>
```

Comment:
As part of the project requirements, kindly make sure you have different components for creating/adding books as well as  the one for displaying & removing books. This will help for easy debugging and maintainability. :+1: